kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/utils/KmHelper.java

```diff
@@ -82,23 +87,26 @@ public void onFailure(Object error) {
 
     public static void setStartNewChat(final Context context, final List<String> agentIds, List<String> botIds, boolean isUnique) {
         final ProgressDialog dialog = new ProgressDialog(context);
-        dialog.setMessage("Creating conversation, please wait...");
+        dialog.setMessage(ApplozicService.getContext(context).getString(R.string.create_conversation_info));
         dialog.setCancelable(false);
         dialog.show();
 
         try {
-            new KmChatBuilder(context).setAgentIds(agentIds).setBotIds(botIds).setSingleChat(isUnique).launchChat(new KmCallback() {
-                @Override
-                public void onSuccess(Object message) {
-                    dialog.dismiss();
-                }
+            new KmConversationBuilder(context).setAgentIds(agentIds)
+                    .setBotIds(botIds)
+                    .setSingleConversation(isUnique)
+                    .launchConversation(new KmCallback() {
+                        @Override
+                        public void onSuccess(Object message) {
+                            dialog.dismiss();
+                        }
 
-                @Override
-                public void onFailure(Object error) {
-                    dialog.dismiss();
-                    Toast.makeText(context, context.getString(R.string.unable_to_create_conversation) + " : " + error, Toast.LENGTH_SHORT).show();
-                }
-            });
+                        @Override
+                        public void onFailure(Object error) {
+                            dialog.dismiss();
+                            Toast.makeText(context, context.getString(io.kommunicate.R.string.unable_to_create_conversation) + " : " + error, Toast.LENGTH_SHORT).show();
```

Comment:
@reytum review if this is intentional here io.kommunicate.R, also we can remove space before :

Reply:
Done